### ISSUE TITLE
Update content (text, links, etc.) for Wichita

### ISSUE THREAD

**Issue by doshitan** [openwichita/openbudgetwichita#2]
Look for anything referencing Oklahoma City or OKC. Need to update them to reference Wichita and Open Wichita.

Seth did a good job in 38c510f7bb0a7a7be7d8998134570253e7cfa620, but there's still a few places around. 

- [x] Oklahoma City still referenced in README
- [x] README build badge is for OKC project 
- [x] The wiki page link under "Fork me!" section is to the OKC wiki (which has good info, but some is specific to OKC, need our own cheatsheet)
- [x] `package.json` is still for the OKC project
- [x] CNAME (and `_src/CNAME`?) is still for OKC project, though that might be a discussion for #3.
- [x] Of course all the data/visualizations are still for OKC at this time, though we can make some easy changes to `_src/{_data, _layout, _who-are-we, _what-we-do, contact, feedback, index}.jade` for contact info and stuff

**Comment by br0xen** [comment]
I'm working on grepping through the repo for some things like their twitter handle and stuff.

**Comment by br0xen** [comment]
- I removed Oklahoma City references in the README.
- I updated the build badge to point to this repos url (if we enable it, right? I haven't used Travis...)
- _src/_data.json: Updated references to Wichita, guessed at a potential URL for the Wichita OpenBudget's home (http://budget.openwichita.com)
- The other 'easy wins' in _src are done.
There still might be references to OKC in filenames, and thus the links to those files, just FYI.
